app/jobs/renew_checkout_job.rb

```diff
@@ -24,7 +24,7 @@ def perform(resource:, item_key:, session_token:)
           renewal_count: checkout.renewal_count,
           due_date: due_date,
           status: checkout.status_human,
-          badge: badge(message: 'Successfully renewed')
+          badge: badge(message: 'Successfully renewed', badge_class: 'success')
```

Comment:
if the default arg for `badge` is `'success'` you don't need to define it here... right?

```suggestion
          badge: badge(message: 'Successfully renewed')
```